app/assets/v2/css/grants/grant.css

```diff
@@ -269,27 +306,59 @@
 	margin-left: -33px;
 }
 
+.partners {
+  background: #0D0764;
+  border-radius: 3px;
+}
+
+.partners h2 {
+  letter-spacing: 3px;
+}
+
+.partners img {
+  border-radius: 50px;
+  width: 4rem;
+}
+
+.partners .nav-tabs {
+  border: none;
+
+}
+
+.partners .nav-link,
+.nav-tabs .nav-line.active  {
+  color: white;
+  cursor: pointer;
+}
+
+.partners .nav-link:hover{
+  border: none;
+}
+
+.nav-tabs .nav-line.active  {
```

Comment:
lets avoid customising already used classed by page since this will break when we decide to put all in a big css

Reply:
Ah I tried doing that but it looks cleaner with this layout for grants